clients/cobol-lsp-vscode-extension/src/test/suite/subroutines.spec.test.ts

```diff
@@ -26,8 +27,12 @@ suite("Integration Test Suite: Subroutines resolving", () => {
     this.timeout(helper.TEST_TIMEOUT);
     await helper.showDocument("CALL.cbl");
     const editor = helper.getEditor("CALL.cbl");
-    const diagnostics = await helper.waitForDiagnostics(editor.document.uri);
-    assert.strictEqual(diagnostics.length, 1);
+
+    await helper.waitFor(
+      () => vscode.languages.getDiagnostics(editor.document.uri).length === 2,
```

Comment:
Should the new diags be suppressed while running these unrelated tests?

Reply:
Why we need to suppress something? If it is easy let's do it, but otherwise I do not see any reason to spend time on it